src/controller/Observable.java

```diff
@@ -0,0 +1,39 @@
+package controller;
+
+import model.IObservable;
+import view.IObserver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Observable implements IObservable {
```

Comment:
You can add a line explanation about this design pattern

Reply:
Already added.

Reply:
V